src/graphql/Dashboard/upcoming-flow-runs.gql

```diff
@@ -15,6 +15,11 @@ query UpcomingFlowRuns($projectId: uuid) {
       id
       name
       schedule
+      environment
+      flow_group {
+        id
+        labels
```

Comment:
This should be a fast lookup on a reasonably "small" table, so I expect this will be OK.

This does make me realize that the API does not provide a source of truth for what the "active" labels are, causing the UI to re-implement the logic.  Making a mental note in case there's an easy way to store this more efficiently.

Reply:
I agree, although Hasura's tendency to replace straight joins with nested loops sometimes makes this difficult to predict.